app/components/custom-fields/form.tsx

```diff
@@ -0,0 +1,132 @@
+import { Form, useNavigation } from "@remix-run/react";
+import { useAtom } from "jotai";
+import { useZorm } from "react-zorm";
+import { z } from "zod";
+import { updateTitleAtom } from "~/atoms/custom-fields.new";
+import { isFormProcessing } from "~/utils";
+import FormRow from "../forms/form-row";
+import Input from "../forms/input";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "../forms/select";
+import { Switch } from "../forms/switch";
+import { Button } from "../shared";
+import { Spinner } from "../shared/spinner";
+
+export const NewCustomFieldFormSchema = z.object({
+  title: z.string().min(2, "Title is required"),
+  helpText: z.string(),
+  type: z.string(),
+  required: z.boolean(),
+});
+
+/** Pass props of the values to be used as default for the form fields */
+interface Props {
+  title?: string;
+  helpText?: string;
+}
+
+export const CustomFieldForm = ({ title, helpText }: Props) => {
+  const navigation = useNavigation();
+  const zo = useZorm("NewQuestionWizardScreen", NewCustomFieldFormSchema);
+  const disabled = isFormProcessing(navigation.state);
+  const fieldTypes = ["Text"];
```

Comment:
This will need to be cleaned up.

Reply:
yes, this all static content will be cleaned up. This was added just to make frontend work and backend integration easier